database/migrations/2023_10_30_025946_create_class_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('class', function (Blueprint $table) {
+            $table->id('class_id');
+            $table->foreignId('subject_id')->constrained(
+                table: 'subject', column: 'subject_id'
+            );
+            $table->year('academic_year');
```

Comment:
I think `academic_year` should just be a string instead of year.

(eg. 'A.Y. 2020-2021' or just '2020-2021')

Reply:
Oh yeah I forgot, thanks!